src/diffusers/models/unet_3d_condition.py

```diff
@@ -129,13 +150,21 @@ def __init__(
         conv_in_kernel = 3
         conv_out_kernel = 3
         conv_in_padding = (conv_in_kernel - 1) // 2
-        self.conv_in = nn.Conv2d(
-            in_channels, block_out_channels[0], kernel_size=conv_in_kernel, padding=conv_in_padding
-        )
 
+        if conv_type == "inflated_conv_3d":
+            self.conv_in = InflatedConv3d(
+                in_channels,
+                block_out_channels[0],
+                kernel_size=conv_in_kernel,
+                padding=(conv_in_padding, conv_in_padding),
+            )
+        else:
+            self.conv_in = nn.Conv2d(
+                in_channels, block_out_channels[0], kernel_size=conv_in_kernel, padding=conv_in_padding
+            )
         # time
         time_embed_dim = block_out_channels[0] * 4
-        self.time_proj = Timesteps(block_out_channels[0], True, 0)
+        self.time_proj = Timesteps(block_out_channels[0], flip_sin_to_cos, freq_shift)
```

Comment:
Think we can leave as is no?

Reply:
Yup, I'll remove the function params as well. Simplifies the signature as well.